File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        numbers = {}
        for i in nums:
            if i in numbers:
                numbers[i] += 1
            else:
                numbers[i] = 1
        k_freq_elements = []
        for n in numbers:
            if numbers[n] >= k:
                k_freq_elements.append(n)
        return k_freq_elements

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        int count = 0;
        Set<Integer> numbers = new HashSet<Integer>();
        for (int n: nums) {
            numbers.add(n);
        }
        for (int n: numbers) {
            int num = n;
            int tempCount = 0;
            while (numbers.contains(num)) {
                num++;
                tempCount++;
            }
            if (tempCount > count) {
                count = tempCount;
            }
        }
        return count;
    }
}
